### PR TITLE
Final refactoring of @processElement to bring new style

### PR DESCRIPTION
**Pull request by shnapz**
Refactored `@processElement` in:
- scio-elasticsearch
- scio-examples
- ...

### CODE REVIEW

**Comment by shnapz** [comment]
Refactored @processElement in:
- scio-extra
- scio-gcp
- scio-redis
- scio-smb
- scio-tensorflow
- scio-test

File: scio-examples/src/main/java/org/apache/beam/examples/complete/TfIdf.java [hunk 1]
```diff
@@ -364,15 +374,17 @@ public void processElement(ProcessContext c) {
               ParDo.of(
                   new DoFn<KV<String, CoGbkResult>, KV<String, KV<URI, Double>>>() {
                     @ProcessElement
-                    public void processElement(ProcessContext c) {
-                      String word = c.element().getKey();
-                      Double df = c.element().getValue().getOnly(dfTag);
+                    public void processElement(
+                        @Element KV<String, CoGbkResult> element,
+                        OutputReceiver<KV<String, KV<URI, Double>>> o) {
```
Comment (RustedBones) [inline scio-examples/src/main/java/org/apache/beam/examples/complete/TfIdf.java:379]: Starting from here `outputReceiver` changed to `o`. Any reason for that ?
I would prefer to have consistent definition for text search.

**Review by clairemcginty — COMMENTED** [review]
LGTM aside from Michel's comment about the inconsistent `outputReceiver` name 👍

File: scio-examples/src/main/java/org/apache/beam/examples/complete/TfIdf.java (hunk 1 above)
Comment (shnapz) [reply scio-examples/src/main/java/org/apache/beam/examples/complete/TfIdf.java:379]: @clairemcginty @RustedBones if you don't mind: I would change it to `o` everywhere. This is the naming I saw in Beam's code samples

Comment (shnapz) [reply scio-examples/src/main/java/org/apache/beam/examples/complete/TfIdf.java:379]: My mistake, they name it `out`:
https://beam.apache.org/documentation/programming-guide/
I will rename accordingly

**Comment by shnapz** [comment]
> LGTM aside from Michel's comment about the inconsistent `outputReceiver` name 👍

Fixed all namings

**Review by RustedBones — APPROVED** [review]

**Pull request merged**